### DISCUSSION TITLE
[FR] Autoriser l'édition d'une facture

### DISCUSSION THREAD

**Discussion by FredVisualOffice** [baimard/gestion#210] [Ideas]
Bonjour,
Une facture n'est pas forcément liée à un devis. On peut avoir des factures récurentes (mensuels par exemple) ou facturer un client au comptoir (achat de marchandises).
De plus une facture peut être partielle, c'est à dire qu'on ne facture qu'une partie du devis (ex : paiement des licences logicielles au démarrage du projet et du service à la fin).
Pour toutes ces raisons une facture doit être éditable indépendamment d'un devis.

**Comment by baimard** [comment]
Bonjour,

Tout d'abord merci de ta contribution.

En suite, Il s'agit d'une gestion simplifiée d'une activité de freelancers ou d'indépendant ou de micro entrepreneur.
Légalement une facture doit être lié à un devis, sinon si c'est au comptoir cela s'appelle une caisse enregistreuse.

"On peut avoir des factures récurentes"
Pour les factures récurrentes, un devis doit quand même être établit indiquant que le paiement est mensuel ou trimestriel ou en deux fois etc. Le logiciel permet d'utiliser le même devis pour différentes facture. Le cas que vous m'indiquez est donc couvert.

'Une facture peut être partielle"
Je prends note de ce cas intéressant et je vais voir pour l'intégrer dans ma ROADMAP.
